docs/tutorials/portal/301/portal-301-5.rst

```diff
@@ -22,6 +22,246 @@ Rubin staff will respond to all questions posted there.
+    :name: portal-301-5-3
+    :alt: A cumulative histogram of number of exposures as a function of expMidptMJD. Values steadily increase with time over a span of 20 days.
+
+    Figure 3: The figure showing the cumulative number of exposures obtained with time.
+
+
+Visit image quality plots
+-------------------------
+
+Derived quantities that characterize the quality of images and their properties are found in the ``CcdVisit`` table.
+
+**4.8. Return to the catalog query interface.**
+Click on the "DP1 Catalogs" tab and then on the "Edit ADQL" button.
+Delete the last query statement.
+
+**4.9. Execute a query on the CcdVisit table.**
+This query retrieves a table of all CcdVists (visit and detector combinations) that were observed of the EDFS field.
+
+.. code-block:: SQL
+
+  SELECT visitId, ra, dec, band, seeing, magLim
+  FROM dp1.CcdVisit
+  WHERE CONTAINS(POINT('ICRS', ra, dec),CIRCLE('ICRS', 59.10, -48.73, 1.0))=1
+  ORDER BY visitId
+
+
+**4.10. View the results.**
+The query returns 2448 results, with the central locations of each detector for each CcdVisit overplotted on the coverage map.
+
+**4.11. Create histograms of seeing and magnitude limit.**
+In the "Active Chart" panel, create two new plots that show a histogram of the ``seeing`` column and a histogram of the ``magLim`` column (the 5-sigma limiting magnitude of each detector image).
+It will look like Figure 4.
+
+.. figure:: images/portal-301-5-4.png
+    :name: portal-301-5-4
+    :alt: A plot showing two histograms. On the left is the distribution of seeing in arcsec, and on the right a histogram of magLim in mag.
+
+    Figure 4: The two histograms showing the distribution of seeing and limiting magnitude over all LSSTComCam detectors and visits, in all bands, in DP1.
+
+
+5. Analyze object photometry
+============================
+
+The ``Object`` table, which contains detections and measurements from the ``deep_coadd`` images.
+
+**5.1. Return to the catalog query interface.**
+Delete the last ADQL statement.
+
+**5.2. Execute a query on the Object table.**
+This query will retrieve the PSF and cModel magnitudes in *g*, *r*, and *i* bands, as well as the ``refExtendedness`` parameter, for objects with SNR>5 measurements in *g* and *r* bands in the EDFS field. Since EDFS data is very deep, there are many detections: update the row limit next to the search button from 50,000 to 500,000 to include all objects in the field.
+
+.. code-block:: SQL
+
+  SELECT coord_ra, coord_dec,
+         g_psfMag, i_psfMag, r_psfMag,
+         g_cModelMag, i_cModelMag, r_cModelMag,
+         g_psfFlux, g_psfFLuxErr,
+         r_psfFlux, r_psfFLuxErr,
+         i_psfFlux, i_psfFLuxErr,
+         refExtendedness
+  FROM dp1.Object
+  WHERE CONTAINS(POINT('ICRS', coord_ra, coord_dec), CIRCLE('ICRS', 59, -49, 1))=1
+        AND g_psfFlux/g_psfFluxErr > 5
+        AND r_psfFlux/r_psfFluxErr > 5
+        AND i_psfFlux/i_psfFluxErr > 5
+
+
+**5.3. Select point-like objects.**
```

Comment:
Typo "to be equal 0" --> "to be equal to 0".

Reply:
Adjusted.